liqwid-script-export/liqwid-script-export.cabal

```diff
@@ -0,0 +1,125 @@
+cabal-version:      3.0
+name:               liqwid-script-export
+version:            2.4.0
+extra-source-files: CHANGELOG.md
+author:             Emily Martins <emi@haskell.fyi>
+license:            Apache-2.0
+
+--------------------------------------------------------------------------------
+-- Common Stanza Declarations
+
+-- Language options, warnings, some options for plutus
+
+common lang
+  ghc-options:
+    -Werror -Wall -Wcompat -Wincomplete-uni-patterns
+    -Wno-unused-do-bind -Wno-partial-type-signatures
+    -Wmissing-export-lists -Wincomplete-record-updates
+    -Wmissing-deriving-strategies -Wno-name-shadowing -Wunused-foralls
+    -fprint-explicit-foralls -fprint-explicit-kinds
+
+  default-extensions:
+    BangPatterns
+    BinaryLiterals
+    ConstraintKinds
+    DataKinds
+    DeriveAnyClass
+    DeriveGeneric
+    DeriveTraversable
+    DerivingVia
+    EmptyCase
+    FlexibleContexts
+    FlexibleInstances
+    GeneralisedNewtypeDeriving
+    HexFloatLiterals
+    ImportQualifiedPost
+    InstanceSigs
+    KindSignatures
+    LambdaCase
+    MultiParamTypeClasses
+    NoFieldSelectors
+    NoStarIsType
+    NumericUnderscores
+    OverloadedLabels
+    OverloadedStrings
+    ScopedTypeVariables
+    StandaloneDeriving
+    TupleSections
+    TypeApplications
+    TypeFamilies
+    TypeOperators
+
+  default-language:   Haskell2010
+
+common deps
+  build-depends:
+    , aeson
+    , aeson-pretty
+    , ansi-terminal
+    , base                   ^>=4.18.1.0
+    , base-compat
+    , base16
+    , bytestring
+    , cache
+    , cardano-binary
+    , clock
+    , containers
+    , data-default
+    , data-default-class
+    , filepath
+    , generics-sop
+    , hashable
+    , http-types
+    , liqwid-plutarch-extra
+    , mtl
+    , optics
+    , optics-th
+    , optparse-applicative
+    , plutarch
+    , plutarch-ledger-api
+    , plutus-core
+    , plutus-ledger-api
+    , plutus-tx
+    , ply-core
+    , ply-plutarch
+    , prettyprinter
+    , recursion-schemes
+    , serialise
+    , servant
+    , servant-server
+    , tagged
+    , template-haskell
+    , text
+    , wai
+    , wai-cors
+    , warp
+
+  mixins:
+
+library
+  import:          lang, deps
+  hs-source-dirs:  src
+  exposed-modules:
+    ScriptExport.API
+    ScriptExport.Export
+    ScriptExport.File
+    ScriptExport.Options
+    ScriptExport.ScriptInfo
+    ScriptExport.Stdout
+    ScriptExport.Types
+
+  other-modules:
+    Codec.Serialise.Orphans
+    Data.Cache.Cached
+
+-- executable export-stat
+--   import:         lang, deps
+--   hs-source-dirs: export-stat
+--   main-is:        Main.hs
+--   build-depends:  liqwid-script-export
+
+-- executable example
+--   import:         lang, deps
+--   hs-source-dirs: example
+--   main-is:        Main.hs
+--   build-depends:  liqwid-script-export
```

Comment:
We should either:
- Remove these altogether (the cabal stanzas and the relevant files)
- Uncomment/test these

Reply:
Uff... I forgot to uncomment these!